validator/manifest.json

```diff
@@ -71880,15 +71504,15 @@
           "issue": [
             {
               "severity": "error",
-              "code": "invalid",
+              "code": "unknown",
               "details": {
                 "coding": [
                   {
                     "system": "http://hl7.org/fhir/dotnet-api-operation-outcome",
-                    "code": "1100"
+                    "code": "-1"
                   }
                 ],
-                "text": "The 'idref' attribute is not declared."
+                "text": "Versions can only be used when the ResourceIdentity is a (relative) url"
```

Comment:
Again this weird -1 error instead of the original error message (which seems to be correct).